docs/Dirb.md

```diff
@@ -1,36 +1,57 @@
-# How to install and use Dirb for Mac OS
+# How to install and use Dirb for OSX
+The main goal of this document is to describe how Dirb can be installed, configured and used on a OSX environment. This tool must only be used in SecDevLabs apps and on systems you have proper authorization.
 
 ## Installation 🔧
 
 All you need to do is follow the commands below, and Dirb will be installed in no time!
 
+First, we need to download the installer from [SourceForge]:
 ```sh
-curl -L https://sourceforge.net/projects/dirb/files/dirb/2.22/dirb222.tar.gz/download -o dirb222.tar.gz
+curl -L https://sourceforge.net/projects/dirb/files/dirb/2.22/dirb222.tar.gz/download -o 
```

Comment:
Typo here! I think `dirb222.tar.gz` should be here but it is inside the next command.

Reply:
Nice catch @rafaveira3 ! Thanks!